toontown/estate/DistributedPhoneAI.py

```diff
@@ -225,7 +225,7 @@ def handleAccount(dclass, fields):
             if not item.getDeliveryTime():
                 retCode = item.recordPurchase(av, optional)
                 deliveryTime = 0
-            elif config.GetBool('want-instant-delivery', False):
+            elif av.instantDelivery == True:
```

Comment:
The config works fine why change it ?

Reply:
Besides it being deprecated (then use ConfigVariableBool)

Reply:
This was changed so that the ~InstaDelivery magic word would function correctly. Without it, invoking ~InstaDelivery does not cause items the invoker orders *after* using the magic word to be instantly delivered, The magic word is currently not programmed to change a config variable, and there currently is no ```want-instant-delivery``` variable in this source. Doing it this way allows for items to be instantly delivered on a per-invoker basis (like with ~SetAlwaysHitCogs and ~Immortal), rather than affecting every player on the server.